windows/jitsi-meet/src/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { render } from 'react-dom';
+
+import {
+    RemoteControl,
+    setupScreenSharingForWindow,
+    setupAlwaysOnTopRender,
+    setupWiFiStats
+} from 'jitsi-meet-electron-utils';
+
+import { jitsiMeetDomain } from '../../../config.js';
+
+/**
+ * Jitsi Meet Window Component
+ */
+class JitsiMeetWindow extends React.Component {
+
+    /**
+     * Render function of component
+     * @return {ScriptElement}
+     */
+    render() {
```

Comment:
We don't need the render method at all.

Reply:
Not defining render method will give an error. Because react executes it.
![render](https://user-images.githubusercontent.com/15872348/40313351-2a17844c-5d33-11e8-9a1b-d76b0cace1b3.png)

Reply:
Doh, I thought it would just return null by default, sigh.